apps/text-embeddings/requirements.txt

```diff
@@ -0,0 +1,5 @@
+torch>=2.0
+open_clip_torch>=2.23
+git+https://github.com/openai/CLIP.git
```

Comment:
Do we need both the packages, when using open_clip_torch

Reply:
Removed